app/src/main/java/com/eveningoutpost/dexdrip/Models/BgReading.java

```diff
@@ -1241,22 +1241,50 @@ public static synchronized BgReading bgReadingInsertLibre2(double calculated_val
         // TODO slope!!
```

Comment:
Does this slope refer to the calibration slope? If yes I believe the TODO could be removed :)